networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/subnets/ValidatorBasedStableSubnetSubscriber.java

```diff
@@ -93,6 +97,7 @@ private Set<SubnetSubscription> adjustNumberOfSubscriptionsToNumberOfValidators(
     final int randomSubnetsPerValidator = ValidatorConstants.RANDOM_SUBNETS_PER_VALIDATOR;
     int totalNumberOfSubscriptions =
         min(ATTESTATION_SUBNET_COUNT, randomSubnetsPerValidator * validatorCount);
+    totalNumberOfSubscriptions = max(totalNumberOfSubscriptions, minimumSubnetSubscriptions);
 
```

Comment:
should really make totalNumberOfSubscriptions final, and clean this bit up...
I'd probably look at making your new definition of `totalNumberOfSubscriptions` be the final, and clean up the interim calculation....
```
final int n = min(ATTESTATION_SUBNET_COUNT, ValidatorConstants.RANDOM_SUBNETS_PER_VALIDATOR * validatorCount);
final int totalNumberOfSubscriptions = max(n, minimumSubnetSubscriptions);
```
I wouldn't call the variable `n` potentially :) but im not sure what a good name would be :)